swagger.yaml

```diff
@@ -437,6 +437,15 @@ definitions:
               mobilePhone:
                 type: string
                 description: E.164 formatted phone number. Capable of SMS communication.
+              ssnStatus:
+                type: string
+                description: SSN status
+                enum:
+                  - valid
+                  - invalid
+                  - vault
+                  - archived
+                  - redacted
```

Comment:
Remember to update this enum to be only valid, invalid, vault.

Reply:
@fernmig sure, you need to refresh the page to view the latest change.